packages/cli-tool/package.json

```diff
@@ -28,6 +28,7 @@
     "inquirer-test": "^2.0.1"
   },
   "devDependencies": {
+    "@nimblehq/eslint-config-nimble": "^2.3.0",
```

Comment:
I was curious why we aren't using `@nimblehq/eslint-config-nimble-react` here?

Reply:
Because we use it to lint the CLI scripts. There is no react in this project ;-) 

For fun, I did the test, and it's quite nice built — if we were using it, we would have the following warning message:

> **Warning:** React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.

Reply:
ah, I get it, I asked this after seeing the inconsistency of spaces around curly braces:
```javascript
import {x} from x;
// with @nimblehq/eslint-config-nimble-react, it becomes(due to [prettier](https://github.dev/nimblehq/eslint-config-nimble/blob/develop/packages/eslint-config-nimble-react/lib/rules/prettier.js))
import { x  }  from  x;
```
maybe we need to bring `prettier` inside `@nimblehq/eslint-config-nimble` for consistency with `prettier`?

Reply:
Nice catch!! Could you add an issue in the https://github.com/nimblehq/eslint-config-nimble repository please? 🙌

Reply:
sure Xavier, I'd do that! Thank you!